setup_instructons/bare_metal_ubuntu_arm64.md

```diff
@@ -5,8 +5,7 @@
 ## Contents
 
 - [1 Install ROS](#1-install-ros)
-- [2 Install STSL](#2-install-stsl)
```

Comment:
Was this removal intentional?

Reply:
No, but the section doesn't exist in the markdown so it should be gone. I fixed the numbering now.